examples/fixed_column_width.html

```diff
@@ -94,12 +94,17 @@
 
         // Get Regular Table element.
         const tableApi = document.getElementsByTagName("regular-table")[0];
+
+        const tableElement = tableApi.querySelector("table");
+
+        const colgroupElement = document.createElement("colgroup");
+        tableElement.insertBefore(colgroupElement, tableElement.firstChild);
```

Comment:
I propose we add `colgroup` directly here instead https://github.com/jpmorganchase/regular-table/blob/master/src/js/table.js#L27 . I would love to eventually see also `<col>` generated by regular-table, using a `view_model.js` like `tbody`, `thead` - and then `auto` and `override` properties internally also use this!

Reply:
Great! i just added it.